### DISCUSSION TITLE
[Console] Projects page - Number of Pods Split Across Two Lines

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#144] [Ideas]
Hi!
In the projects page when the number of pods reaches 3 digits the page incorrectly splits it across two separate lines as shown here:

![image](https://user-images.githubusercontent.com/21307773/208706075-6e1b6018-c52c-4e89-98ef-2c72355712e0.png)

I would expect the console to either switch to a smaller font size or to separate the line at a more appropriate location (i.e. after the `/`), as shown below:

![image](https://user-images.githubusercontent.com/21307773/208706904-a90d2937-5997-49f3-a0f0-dbdf673b1fc7.png)
![image](https://user-images.githubusercontent.com/21307773/208707122-6be8faa0-e3c6-4013-af68-a59919c86c4e.png)

**Comment by fredmaggiowski** [comment]
Thank you for your feedback, it seems like a ui bug, @FrankieBortot what do you think about it? I'd rather the first solution with an adaptive font size

**Reply by FrankieBortot** [reply]
Hi there!
I would suggest using a resizable font size

**Reply by fredmaggiowski** [reply]
@albertotessarotto FYI we should track this small improvement in our internal backlog

**Reply by albertotessarotto** [reply]
yeah, already already tracked the activity in our backlog
